test/runtime-integration/src/main.mjs

```diff
@@ -0,0 +1,3 @@
+import "./import-auto.mjs";
+import "./import-esm.mjs";
+import "./require-auto.cjs";
```

Comment:
Can you add a case on `./require-esm.cjs`?

Reply:
Should it work or throw?

Reply:
It should throw on Node. But other bundlers, especially legacy versions may have different behaviour.